src/complate/bundling.js

```diff
@@ -14,7 +14,7 @@ export default class VirtualBundle extends Bundle {
 			parser: acornJSX()
 		});
 		config.addPlugin(new Plugin("diskless", dl));
-		config.addPlugin(new Plugin("complate-rodunj", rodunj));
+		config.addPlugin(new Plugin("complate-ast", complate, { prefix: "__gorgeon_" }));
```

Comment:
This prefix appearing in two places seems like a smell. It's not immediately obvious to me how this interacts with `renderString` below.